Documentation/install/upgrade.rst

```diff
@@ -7,7 +7,14 @@ Upgrade Guide
 Kubernetes Cilium Upgrade
 =========================
 
-Cilium should be upgraded using Kubernetes rolling upgrade functionality in order to minimize network disruptions for running workloads. 
+Cilium should be upgraded using Kubernetes rolling upgrade functionality in order to minimize network disruptions for running workloads.
+
+If you are using RBAC in your cluster make sure you are running the latest RBAC
```

Comment:
you are using the latest RBAC role and service account definitions before performing the rolling upgrade:

Reply:
Done